### PR TITLE
Request Missed Headers in Powchain Service

### PR DESCRIPTION
**Pull request by nisdas**
This checks if any headers have been missed and requests them from the powchain

### CODE REVIEW

File: beacon-chain/powchain/service.go [hunk 1]
```diff
@@ -297,6 +298,27 @@ func (w *Web3Service) handleDelayTicker() {
 	}
 }
 
+func (w *Web3Service) handleSkippedHeaders(ctx context.Context, header *gethTypes.Header) {
+	headerHeight := header.Number.Uint64()
+	currentHeight := w.blockHeight.Uint64()
+
+	if headerHeight > currentHeight+1 {
```
Comment (terencechain) [inline beacon-chain/powchain/service.go]: Do we need this if condition. The for loop should just take care of this right?

Comment (nisdas) [reply beacon-chain/powchain/service.go]: yeah the condition is redundant in this case, removed it

**Comment by terencechain** [comment]
travis failed

`beacon-chain/main.go:97:25: undefined: log.FluentdFormatter`

**Comment by terencechain** [comment]
> travis failed
> 
> `beacon-chain/main.go:97:25: undefined: log.FluentdFormatter`

Nevermind, it's a general failure happening in other PRs. I can fix it

**Pull request closed without merging**